examples/some-issuer/json-schemas/discord/credential-metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "title": "Discord Certificate",
+  "logo": {
+    "url": "https://avatars.githubusercontent.com/u/39614219?s=200&v=4"
+  },
+  "background_color": "#7289da"
```

Remember to change this

```suggestion
  "backgroundColor": "#7289da"
```